File: ButterKnifeSample/app/src/main/java/com/example/butterknifesample/MainActivity.java
package com.example.butterknifesample;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.Button;
import android.widget.RadioButton;

public class MainActivity extends AppCompatActivity {

    private RadioButton exampleRadioButton;
    private Button exampleButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        exampleRadioButton = (RadioButton) findViewById(R.id.exampleRadioButton);
        exampleButton = (Button) findViewById(R.id.exampleButton);

    }
}

File: LifecycleSample001/app/src/main/java/com/example/iyemon018/lifecyclesample001/MainActivity.kt
package com.example.iyemon018.lifecyclesample001

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    /**
     * 初期化処理やビューの作成などを行う
     */
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
